### DISCUSSION TITLE
backup

### DISCUSSION THREAD

**Discussion by vasnt** [teableio/teable#509] [Q&A]
How to backup, export teable database?

**Comment by tea-artist** [comment]
<img width="304" alt="image" src="https://github.com/teableio/teable/assets/113977203/8d91a428-ec2a-4440-b5d6-1be62c7a4351">

export to csv

or use pg_dump after create a database connection

`pg_dump -U yourusername -d yourdbname -n yourschema > outputfile.sql`

**Reply by vasnt** [reply]
Thanks @tea-artist i understand
- pg-dump is cli based solution. (i prefer if any gui/visual way out)
- export to csv is manual activity instead of automatic backup
- Can https://dbeaver.io/ or alternative solve this issue?
